locals.tf

```diff
@@ -1 +1,8 @@
 # TODO: insert locals here.
+
```

Comment:
enable_telemetry is already defined as a variable with default value true in variable.tf, you would need to updated main.telemetry.tf line 2 accordingly to use var.enable_telemetry instead and I see that locals.telemetry.tf hasn't been updated